### PR TITLE
Tweak tests documentation

### PR DESCRIPTION
**Pull request by jkonecny12**
Make more obvious that this is about how to run our tests not how to debug Anaconda.

### CODE REVIEW

File: tests/README.rst [hunk 1]
```diff
@@ -1,10 +1,18 @@
 Testing Anaconda
 ================
 
-Testing locally
----------------
+This document describes how to run Anaconda tests. Anaconda has various tests for example
```
Comment (M4rtinK) [inline tests/README.rst]: I suggest "such as" instead of "for example" here.

File: tests/README.rst [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+unit tests, rpm tests, translation tests. All the tests will be run together if you follow
```
Comment (M4rtinK) [inline tests/README.rst]: I suggest:
"rpm tests, translation tests." -> "rpm tests or translation tests."

File: tests/README.rst [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
+steps below.
```
Comment (M4rtinK) [inline tests/README.rst]: I suggest:
"follow the steps below."

File: tests/README.rst [hunk 4]
```diff
@@ -1,10 +1,18 @@
...
 
-Before testing Anaconda you need to install all required dependencies.
+You have two alternate ways how to run these tests. First is directly on your system or
```
Comment (M4rtinK) [inline tests/README.rst]: I suggest using bullet points for the two way:

- running the test directly on your system
- using mock utility which run the tests in a container on your system

Read bellow about benefits and drawbacks of the two methods.

**Comment by poncovka** [comment]
Jenkins, test this please.

File: tests/README.rst (hunk 2 above)
Comment (jkonecny12) [reply tests/README.rst]: It should be `and` but yes I'll change that.

File: tests/README.rst (hunk 4 above)
Comment (jkonecny12) [reply tests/README.rst]: Nice improvement. Thanks!

**Comment by jkonecny12** [comment]
**UPDATED:**
* Implemented suggestions from @M4rtinK - thanks a lot!

**Comment by jkonecny12** [comment]
jenkins, test this please

**Comment by jkonecny12** [comment]
jenkins, test this please

**Comment by M4rtinK** [comment]
@jkonecny12 np :-)

**Pull request merged**